doc/AUTHORS.md

```diff
@@ -166,3 +166,11 @@ documentation, Python bindings for elektraMerge, copy-on-write functionality of
 - email: max@maxirlinger.at
 - GitHub user: [atmaxinger](https://github.com/atmaxinger)
 - devel/test on: Fedora, macOS
+
+## Florian Lindner
+
+mounting library, [unixODBC](https://www.unixodbc.org) backend
```

Comment:
Is mounting its own library? If so, maybe add this information also in a README.md for that library.

Reply:
Libraries currently don't list authors in the README

Reply:
[They soon will](https://github.com/ElektraInitiative/libelektra/pull/4866#discussion_r1133216781) 😉